app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    Cafe.count_friends = 0
```

Comment:
Using `Cafe.count_friends` as a class attribute and resetting it here can lead to issues if multiple instances of `Cafe` are used concurrently. Consider using an instance attribute instead.